src/test/java/com/vinyl/RemoveProductFromStoreTest.java

```diff
@@ -0,0 +1,95 @@
+package com.vinyl;
+
+import com.vinyl.helper.TokenHeaderHelper;
+import com.vinyl.model.Product;
+import com.vinyl.repository.ProductRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+
+import java.time.LocalDate;
+
+public class RemoveProductFromStoreTest extends ManagerBaseIntegration {
+
+    @Autowired
+    protected TokenHeaderHelper tokenHeaderHelper;
```

Comment:
private fields since they are used only in this class

Reply:
fixed